src/zenml/cli/utils.py

```diff
@@ -212,7 +208,7 @@ def print_stack_component_list(
 ) -> None:
     """Prints a table with configuration options for a list of stack components.
 
-    If a component is active (its name matches the `active_component_name`),
+    If a component is active(its name matches the `active_component_name`),
```

Comment:
Was this whitespace removal intentional (also happened in some other place further down)?